client/src/locale/pt-PT.locale.ts

```diff
@@ -0,0 +1,133 @@
+    home: {
+        title: 'Pacstall',
+        subtitle: 'O AUR para Ubuntu',
+        cards: {
+            whyDifferent: {
+                title: 'Porque é que isto é diferente de qualquer outro gestor de pacotes?',
+                description:
+                    'O Pacstall usa a base estável do Ubuntu mas permite-lhe usar o ' +
+                    'software de última geração com poucos ou nenhuns compromissos, pelo que ' +
+                    'não tem de se preocupar com correções de segurança ou novas funcionalidades.',
+            },
+            howItWorks: {
+                title: 'Como funciona então?',
+                description:
+                    ' O Pacstall recebe ficheiros chamados de <0>pacscripts</0> (semelhantes ' +
+                    ' ao PKGBUILD) que contêm a informação necessária para compilar pacotes ' +
+                    ' e compila-os como executáveis no seu sistema.',
+            },
+        },
+        installationInstructions: 'Instruções de instalação',
+        showcase: {
+            title: 'Demonstração',
+            packageSearch: 'Procurar pacotes',
+        },
+    },
+    navbar: {
+        title: 'Pacstall',
+        contribute: {
+            title: 'Contribua',
+            workOnFeatures: 'Trabalhe em novas funcionalidades',
+            helpTranslate: 'Ajude com as traduções',
+            becomeAMaintainer: 'Torne-se um responsável de pacotes',
+        },
+        social: {
+            title: 'Redes sociais',
+            discord: 'Discord',
+            matrix: 'Matrix',
+            reddit: 'Reddit',
+            lemmy: 'Lemmy',
+            mastodon: 'Mastodon',
+        },
+        browse: {
+            title: 'Navegar pelos pacotes',
+        },
+        privacy: {
+            title: 'Poliítica de privacidade',
+        },
+        install: 'Instalar',
+    },
+    cookieConsent: {
+        title: 'Aviso sobre Cookies (TL;DR)',
+        paragraphs: [
+            'Olá, estamos a utilizar cookies na nosso website.',
+            'Não gostamos de lhe dar informações enganadoras ou confusas. Apenas utilizamos cookies para funcionalidades essenciais, como definições de temas, localização e autenticação.',
+            'Pode ler a política de privacidade completa <0>aqui <1/></0>.',
+            'Ao continuar a utilizar este site, está a <strong>concordar com a política de privacidade</strong>.',
+        ],
+        accept: 'Aceitar',
+    },
+    packageSearch: {
+        dropdown: {
+            package: 'Pacote',
+            packageTooltip: 'Procurar em nomes e descrições de pacotes',
+            maintainer: 'Responsável',
+            maintainerTooltip: 'Procurar por nomes e e-mails de responsáveis',
+        },
+        table: {
+            name: 'Nome',
+            maintainer: 'Responsável',
+            version: 'Versão',
+            install: 'Instalar',
+        },
+        versionTooltip: {
+            notInRegistry: 'Este pacote não está registado no Repology',
```

Comment:
```suggestion
            notInRegistry: 'Este pacote não está registrado no Repology',
```

Reply:
> 

The word **registrado** doesn't exist in the portuguese dictionary! The correct word is **registado**. In Brazil, they use the word **registrado** with the "r", but it's still a word invented by Brazilians.

Reply:
Ambas grafias estão corretas,correspondendo ao particípio do verbo registrar/registar (algo que eu não sabia, e um  fato que considero curioso).

E sobre a palavra ter sido "inventada", isso não tem embasamento linguístico. A origem da palavra é do latim [registrum](https://en.wiktionary.org/wiki/registrum#Declension). Sendo assim, a forma portuguesa da palavra foi a que sofreu aférese.

Reply:
Sendo esta uma tradução do Português de Portugal, a palavra registo é a que está correcta,

Reply:
Sim, claro, eu concordo